x-pack/winlogbeat/module/security/config/winlogbeat-security.js

```diff
@@ -47,56 +157,46 @@ var security = (function () {
         target: "",
     });
 
-    var convertAuthentication = new processor.Convert({
-        fields: [
-            {from: "winlog.event_data.TargetUserSid", to: "user.id"},
-            {from: "winlog.event_data.TargetUserName", to: "user.name"},
-            {from: "winlog.event_data.TargetDomainName", to: "user.domain"},
-            {from: "winlog.event_data.ProcessId", to: "process.pid", type: "long"},
-            {from: "winlog.event_data.ProcessName", to: "process.executable"},
-            {from: "winlog.event_data.IpAddress", to: "source.ip", type: "ip"},
-            {from: "winlog.event_data.IpPort", to: "source.port", type: "long"},
-            {from: "winlog.event_data.WorkstationName", to: "source.domain"},
-        ],
-        mode: "rename",
-        ignore_missing: true,
-        fail_on_error: false,
-    });
-
-    var setProcessNameUsingExe = function(evt) {
-        var name = evt.Get("process.name");
-        if (name) {
-            return;
-        }
-        var exe = evt.Get("process.executable");
-        evt.Put("process.name", path.basename(exe));
-    };
-
+    // Handles 4634 and 4647.
     var logoff = new processor.Chain()
-        .Add(convertAuthentication)
+        .Add(copyTargetUser)
```

Comment:
If anyone tries to run this pipeline on the existing 7.2.0 release you will need to change this line to add `.Run` like

`.Add(copyTargetUser.Run)`

and do the same for `.Add(copySubjectUser)` and `.Add(renameCommonAuthFields)`